.rulesets/ApplicationInsightsSDKRules.ruleset

```diff
@@ -31,7 +31,13 @@
     <Rule Id="CA1410" Action="Warning" />
     <!-- Suppressing Message "Identifiers should not match keywords" because class TplActivities is part of our public api and can't be changed -->
     <Rule Id="CA1716" Action="None" />
+    <!-- Errors shown after FxCop upgrade to 3.3.0. Turning off CA1805 to ignore these errors-->
```

Comment:
This comment doesn't tell me what the rule is or why it was disabled.
See the comment on line 32